src/encode.rs

```diff
@@ -22,6 +22,9 @@ impl From<f32> for OrePlaintext<u64> {
 
 impl From<f64> for OrePlaintext<u64> {
     fn from(term: f64) -> OrePlaintext<u64> {
+        if term == 0f64 || term == -0f64 {
+            return OrePlaintext(0u64);
+        }
```

Comment:
I *think* this means we will end up with an unstable sort order for records indexed on a float when we do "order by" in queries in the presence of both 0 and -0. Essentially, the order that records come back will be non-deterministic.

Reply:
When 0 and -0 are massaged into a u64, what were the two u64 representations? I'm guessing that they were not adjacent and hence the weird comparison result.

Reply:
Actually, I'm ok with this. It looks like I was already going down this path somewhat but didn't actually fix the encoding.